### PR TITLE
fix(notifier): uses subscription expiration instead of plan expiration

### PR DESCRIPTION
**Pull request by vjgene**
https://rsklabs.atlassian.net/browse/RMKT-798

https://user-images.githubusercontent.com/11134695/124575825-03f1ef80-de4c-11eb-8717-fcbd1a85707c.mp4

### CODE REVIEW

File: src/components/molecules/ExpirationDate.tsx [hunk 1]
```diff
@@ -15,7 +15,7 @@ const useStyles = makeStyles(({ palette }: Theme) => ({
     color: palette.warning.main,
   },
   blocked: {
-    color: palette.text.disabled,
+    color: palette.error.main,
```
Comment (jurajpiar) [inline src/components/molecules/ExpirationDate.tsx:18]: I think it has been agreed to ditch the red colouring in favour of disabling it, unless that requirement had changed recently. @artem-brazhnikov could you confirm either way, please?

**Review by jurajpiar — DISMISSED** [review]
let's check with @artem-brazhnikov about the ux

File: src/components/molecules/ExpirationDate.tsx (hunk 1 above)
Comment (0xartem) [reply src/components/molecules/ExpirationDate.tsx:18]: @jurajpiar it should be disabled and grey if the plan was deleted. and red if expired

File: src/components/pages/notifier/mypurchase/mapMyPurchases.tsx [hunk 2]
```diff
@@ -7,15 +7,17 @@ import { NotifierSubscriptionItem } from 'models/marketItems/NotifierItem'
...
+import { SUBSCRIPTION_STATUS } from 'api/rif-marketplace-cache/notifier/subscriptions/models'
 
 const EXPIRATION_WARNING_TRIGGER = 5
 
 const getExpirationType = (
-  { planStatus, daysLeft }: PlanDTO,
+  { status, expirationDate }: Pick<NotifierSubscriptionItem, 'status' | 'expirationDate'>,
 ): SubscriptionExpirationType => {
-  if (planStatus !== PLAN_STATUS.ACTIVE || daysLeft <= 0) return 'blocked'
+  const daysLeft = Math.ceil(Math.abs(new Date().getTime() - expirationDate?.getTime()) / 86400000)
```
Comment (itofarina) [inline src/components/pages/notifier/mypurchase/mapMyPurchases.tsx]:
```suggestion
  const daysLeft = Math.ceil(Math.abs(new Date().getTime() - expirationDate?.getTime()) / 60*60*24*1000)
```

Or create a util function `timestampToDays(milliseconds)`

**Review by itofarina — APPROVED** [review]

**Pull request merged**